index.html

```diff
@@ -1,402 +1,558 @@
-							data-bs-toggle="modal"
-							data-bs-target="#loginModal"
-							data-auth="login"
-							data-visible="loggedOut"
-						>
-							Login
-						</button>
-						<button type="submit" class="btn btn-success">Create Profile</button>
-					</div>
-				</form>
-			</div>
-		</div>
+    <div
+      class="modal fade"
+      id="registerModal"
+      data-bs-backdrop="static"
+      data-bs-keyboard="false"
+      tabindex="-1"
+      aria-labelledby="registerModalLabel"
+      aria-hidden="true"
+    >
+      <div
+        class="modal-dialog modal-dialog modal-dialog-centered modal-dialog-scrollable"
+      >
+        <form class="modal-content" id="registerForm">
+          <div class="modal-header">
+            <h5 class="modal-title" id="registerModalLabel">Create Profile</h5>
+            <button
+              type="button"
+              class="btn-close"
+              data-bs-dismiss="modal"
+              aria-label="Close"
+            ></button>
+          </div>
+          <div class="modal-body">
+            <div class="form-floating mb-3">
+              <input
+                required
+                type="text"
+                class="form-control"
+                id="registerName"
+                name="name"
+                placeholder="name@example.com"
+              />
+              <label for="registerName">Name</label>
+            </div>
+            <div class="form-floating mb-3">
+              <input
+                required
+                type="email"
+                class="form-control"
+                id="registerEmail"
+                name="email"
+                pattern="[\w\-.]+@(stud.)?noroff.no$"
+                title="Only Noroff student or teacher emails are valid."
+                placeholder="name@example.com"
+              />
+              <label for="registerEmail">Email address</label>
+            </div>
+            <div class="form-floating mb-3">
+              <input
+                required
+                type="password"
+                minlength="8"
+                class="form-control"
+                id="registerPassword"
+                name="password"
+                placeholder="Password"
+              />
+              <label for="registerPassword">Password</label>
+            </div>
+            <div class="form-floating">
+              <input
+                required
+                type="url"
+                class="form-control"
+                id="registerAvatar"
+                name="avatar"
+                placeholder="Avatar URL"
+              />
+              <label for="registerAvatar">Avatar URL</label>
+            </div>
+          </div>
+          <div class="modal-footer">
+            <button
+              type="reset"
+              class="btn btn-secondary"
+              data-bs-dismiss="modal"
+            >
+              Close
+            </button>
+            <button
+              id="pls"
```

Comment:
You should not be editing the original code, find a different way to target this button